Providers/Pocket/PocketSponsoredStoriesProvider.swift

```diff
@@ -0,0 +1,127 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Shared
+
+class PocketSponsoredStoriesProvider: PocketSponsoredStoriesProviding, FeatureFlaggable, URLCaching, Loggable {
+
+    var endpoint: URL {
+        if featureFlags.isCoreFeatureEnabled(.useStagingSponsoredPocketStoriesAPI) {
+            return PocketSponsoredConstants.staging
+        } else {
+            return PocketSponsoredConstants.prod
+        }
+    }
+
+    lazy var urlSession: URLSession = makeURLSession(userAgent: UserAgent.defaultClientUserAgent, configuration: URLSessionConfiguration.default)
+    private lazy var pocketKey: String = {
+        return Bundle.main.object(forInfoDictionaryKey: PocketSponsoredConstants.pocketEnvAPIKey) as? String ?? ""
+    }()
+    lazy var urlCache: URLCache = {
+        return URLCache.shared
+    }()
```

Comment:
```suggestion
    lazy var urlSession: URLSession = makeURLSession(userAgent: UserAgent.defaultClientUserAgent, configuration: URLSessionConfiguration.default)

    private lazy var pocketKey: String = {
        return Bundle.main.object(forInfoDictionaryKey: PocketSponsoredConstants.pocketEnvAPIKey) as? String ?? ""
    }()

    lazy var urlCache: URLCache = {
        return URLCache.shared
    }()
```
Nit. You have spaces between some things, and not others. Generally we prefer spaces, but, consistency is the most important. haha